node/src/test_utils/automap_mocks.rs

```diff
@@ -57,6 +54,7 @@ impl AutomapControlFactoryMock {
     }
 }
 
+#[derive(Clone)]
```

Comment:
Cloning mocks tastes fishy. Do you really need that? I always try as hard as possible to avoid this. 

If you're cloning it some of your tests, maybe a scenario test, you could have a closure producing this, that you could call multiple times to produce the same subject any times you need.

Reply:
Nope, don't need it.